app/src/main/res/layout/activity_main.xml

```diff
@@ -15,5 +15,34 @@
             app:layout_constraintLeft_toLeftOf="parent"
             app:layout_constraintRight_toRightOf="parent"
             app:layout_constraintTop_toTopOf="parent" android:id="@+id/textView"/>
+    <Button
+            android:text="@string/button1"
```

Comment:
Do you see what happened with this button in landscape orientation?)

![image](https://user-images.githubusercontent.com/16294951/54492677-b0a1af00-48d9-11e9-9ccc-62e5788571d6.png)
You need to make something like this
![image](https://user-images.githubusercontent.com/16294951/54492696-d29b3180-48d9-11e9-8618-306d207daf64.png)

Reply:
@egorikftp done